src/gutenberg2zim/export.py

```diff
@@ -744,11 +743,15 @@ def optimize_epub(src, dst):
         # delete {id}/cover.jpg if exist and update {id}/content.opf
         if remove_cover:
             # remove cover
-            Path(os.path.join(tmpd, text_type(book.id), "cover.jpg")).unlink_p()
+            file_path = Path(tmpd) / text_type(book.id) / "cover.jpg"
+            try:
+                file_path.unlink()
+            except FileNotFoundError:
+                pass
 
             soup = None
-            opff = os.path.join(tmpd, text_type(book.id), "content.opf")
-            if os.path.exists(opff):
+            opff = Path(tmpd) / text_type(book.id) / "content.opf"
+            if Path(opff).exists():
                 opff_content, _ = read_file(opff)
```

Comment:
opff is already a Path
```suggestion
            if opff.exists():
```